lib/src/conflicts.rs

```diff
@@ -162,6 +164,7 @@ impl MaterializedFileValue {
         let mut buf = Vec::new();
         self.reader
             .read_to_end(&mut buf)
+            .block_on()
```

Comment:
nit: this can be changed to `async fn` (can be addressed later)

Reply:
True. I'll do that later.